### ISSUE TITLE
Error: undefined - undefined for control widget on customer dashboard

### ISSUE THREAD

**Issue by iDeam0316** [thingsboard/thingsboard#1472]
I added a control widget on the dashboard. It worked fine when I login as Tenant administrator. 

However, when I login as a customer, the control widget appeared "Error: undefined - undefined". And it was no longer work on the customer dashboard.

**Comment by vparomskiy** [comment]
did you assign this device to the Customer?
https://thingsboard.io/docs/user-guide/ui/devices/#assign-devices-to-customers
https://thingsboard.io/docs/user-guide/rbac/

**Comment by iDeam0316** [comment]
Yes, the device is assigned to the customer. As customer dashboard is able to view the timeseries chart. But not the control widget, like the switch widget

**Comment by haydenmkoch** [comment]
Any progress?

**Comment by ikulikov** [comment]
The issue was fixed recently in master branch. Fix will be available in v.
2.3.1.
